pytest_tests/testsuites/network/test_node_management.py

```diff
@@ -89,6 +89,18 @@ def return_nodes_after_test_run(client_shell: Shell, hosting: Hosting):
     return_nodes(client_shell, hosting)
 
 
+@allure.step("Tick epoch with retries")
+def tick_epoch_with_retries(shell: Shell, attempts: int = 3, timeout: int = 3):
```

Comment:
New decorator is available from test_control
wait_for_success - it has timeout and interval between checks, so it could like like this
```
@wait_for_success(max_wait_time=10, interval=3)
def tick_epoch_with_timeout(...)
      tick_epoch(...)
```